### PR TITLE
do not know what is wrong with tests

### PR DESCRIPTION
**Pull request by NatalyKrvch**
[DEMO LINK](https://NatalyKrvch.github.io/layout_stars/)
[TEST REPORT LINK](https://NatalyKrvch.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(-n+1)::before,
+.stars--2 .stars__star:nth-child(-n+2)::before,
+.stars--3 .stars__star:nth-child(-n+3)::before,
+.stars--4 .stars__star:nth-child(-n+4)::before,
+.stars--5 .stars__star:nth-child(-n+5)::before {
```
Comment (etojeDenys) [inline src/style.css:30]: why do you need the before here, it looks redundant
```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 1 above)
Comment (NatalyKrvch) [reply src/style.css:30]: @etojeDenys  after implementing your suggestion stars look like this
Capture

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
- please check this video and try to pass the tests 

https://github.com/mate-academy/layout_stars/assets/64746943/bdbc71eb-3c91-4b44-af4e-0b6fb5814570

Screenshot 2023-10-02 at 13 06 32

File: src/index.html [hunk 2]
```diff
@@ -7,7 +7,58 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+
+    <div class="container">
+
+      <div class="stars stars--0">
```
Comment (Viktor-Kost) [inline src/index.html:14]: no needs to add an empty line between parent tag and child one
```suggestion
  <body class="body">
    <div class="container">
      <div class="stars stars--0">
```

File: src/index.html [hunk 3]
```diff
@@ -7,7 +7,58 @@
...
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars stars--5">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+
```
Comment (Viktor-Kost) [inline src/index.html:60]:
```suggestion
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
```
Comment (Viktor-Kost) [inline src/style.css:22]: last star shouldn't have margin. use `:not(:last-child)`

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,31 @@
+.body {
```
Comment (Viktor-Kost) [inline src/style.css:1]: usually we don't use class selector for `body` and `html` tag. we use tag selectors directly

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Nice!
Some minor fixes needed

File: src/style.css (hunk 4 above)
Comment (NatalyKrvch) [reply src/style.css:22]: @Viktor-Kost didn't get it. where exactly should I use this? I tried and did not succeed

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--5:not(:last-child) {
```
Comment (Viktor-Kost) [inline src/style.css:25]:
```suggestion
.stars__star:not(:last-child) {
```

**Review by Viktor-Kost — APPROVED** [review]
Nice!)
Just pay attention at comment I left for you

**Pull request closed without merging**